ledger-api/grpc-definitions/com/daml/ledger/api/v1/admin/user_management_service.proto

```diff
@@ -104,13 +104,23 @@ message CreateUserRequest {
     repeated Right rights = 2;
 }
 
+message CreateUserResponse {
+    // Created user.
+    User user = 1;
```

Comment:
We normally document which field is required and which is optional. I have realized that it is missing in this file throughout. I have added a JIRA ticket to fix that

Reply:
This should be fixed in another PR comprehensively.